### PR TITLE
add some comments

### PR DESCRIPTION
**Pull request by cangfengzhs**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### What problem(s) does this PR solve?
Issue(s) number: 

Description:

#### Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/storage/exec/QueryUtils.h [hunk 1]
```diff
@@ -44,7 +50,14 @@ class QueryUtils final {
       return ReturnColType::kOther;
     }
   }
-
+  /**
+   * @brief Get value with propName from reader
+   *
+   * @param reader Value set
```
Comment (jackwener) [inline src/storage/exec/QueryUtils.h:56]: I don,t recommend this style of comment. It,s redundant.

File: src/storage/exec/RelNode.h [hunk 2]
```diff
@@ -27,13 +27,22 @@ using PropHandler = std::function<nebula::cpp2::ErrorCode(
 template <typename T>
 class StoragePlan;
 
-// RelNode is shortcut for relational algebra node, each RelNode has an execute
-// method, which will be invoked in dag when all its dependencies have finished
+/**
+ * @brief RelNode is shortcut for relational algebra node, each RelNode has an execute method, which
```
Comment (pengweisong) [inline src/storage/exec/RelNode.h:31]: Abbreviation is more suitable?

File: src/storage/exec/RelNode.h [hunk 3]
```diff
@@ -27,13 +27,22 @@ using PropHandler = std::function<nebula::cpp2::ErrorCode(
...
+ *
+ * @tparam T is input data type of plan
+ */
 template <typename T>
 class RelNode {
   friend class StoragePlan<T>;
 
  public:
+  /**
+   * @brief start execution with `input` and `partId`
```
Comment (pengweisong) [inline src/storage/exec/RelNode.h:42]: What's the different with execute and doExecute

File: src/storage/exec/UpdateResultNode.h [hunk 4]
```diff
@@ -66,7 +84,10 @@ class UpdateResNode : public RelNode<T> {
   std::vector<Expression*> returnPropsExp_;
   StorageExpressionContext* expCtx_;
 
-  // return prop sets
+  /**
```
Comment (SuperYoko) [inline src/storage/exec/UpdateResultNode.h:87]: Don't think it's necessary.

Comment (cangfengzhs) [reply src/storage/exec/UpdateResultNode.h:87]: Just for the sake of uniform formatting

**Review by SuperYoko — APPROVED** [review]

**Review by pengweisong — APPROVED** [review]

**Pull request merged**